### DISCUSSION TITLE
Not able to access tutorials and examples from Jupyter Notebook

### DISCUSSION THREAD

**Discussion by cejoydean** [IDAES/idaes-pse#1272] [General]
Hello,

I am completely new to IDAES. have been trying to access the IDAES tutorials to improve my experience with it. I followed every step in the Install article. 

In the Anaconda environment, the command "idaesx gui" does not open the Jupyter interface, because I get an error that in the "idaes_examples/build.py" there is no module named "jupyter cache."

I am able to access Jupyter by entering "jupyter notebook" into Anaconda, and find the idaes-examples folder deep within my personal files. However, following along the [ Youtube tutorial ](https://youtu.be/_E1H4C-hy14?t=946), I see that opening Jupyter should bring me to a "examples" folder which has guided tutorials with solutions. I realize that the Youtube tutorial is a few years old. Do these tutorials still exist? Any idea why I am not able to access them as such? 

Thank you,

**Comment by lbianchi-lbl** [comment]
Hi @cejoydean, welcome to IDAES, and thanks for bringing this to our attention. We can confirm that both problems need to be addressed on our side, and I've created two separate GitHub issues to keep track of the progress: IDAES/examples#73 and IDAES/examples#74.

In the meantime, can you try running the following command in the same environment where IDAES is installed? This should install the extra dependencies needed for the examples GUI browser to work:

```
pip install "idaes-examples[gui]"
idaesx gui
```

**Reply by cejoydean** [reply]
Yes, this worked! Thank you

**Reply by lbianchi-lbl** [reply]
@cejoydean glad to hear it helped. FYI, I've just opened a small PR adding that information to the IDAES/examples README, so that our suggested workflow works as intended for future users: https://github.com/IDAES/examples/pull/75
